### PR TITLE
Notifications Banner

### PR DESCRIPTION
**Pull request by coenraadf**
Add CMS driven component to site, to be able to raise notifications at the top of the site.

- allow for a boolean toggle to turn on and off
- allow for Markdown content to be added
- allow for text-color, background-color and hover-color to be set

Screenshot 2020-09-21 at 15 27 50

### CODE REVIEW

**Comment by dewetvdm** [comment]
Mornings! What is this little tag here?

Image 2020-09-22 at 9 24 35 AM

File: src/components/NotificationsBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import get from 'lodash/get'
+import styled from '@emotion/styled'
```
Comment (dewetvdm) [inline src/components/NotificationsBanner.tsx:4]: I see there are q couple of linting warnings, do you mind cleaning them up please?

File: src/components/NotificationsBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import {
+  unstable_Form as Form,
+  unstable_FormLabel,
+  unstable_FormInput,
+  unstable_FormMessage,
+  unstable_FormSubmitButton as FormSubmitButton
+} from 'reakit/Form'
+import { Section, SectionContainer, SectionTitle } from './Section'
+import Button from './Button'
+import { useContactUsForm } from '../hooks/contact-us-form'
```
Comment (dewetvdm) [inline src/components/NotificationsBanner.tsx:18]: also don't think we need this hook here

File: src/components/NotificationsBanner.tsx (hunk 1 above)
Comment (coenraadf) [reply src/components/NotificationsBanner.tsx]: fixed. missed those.

**Comment by coenraadf** [comment]
> Mornings! What is this little tag here?
> 
> Image 2020-09-22 at 9 24 35 AM

this is just something from the screenshot. not sure where it comes form

**Comment by coenraadf** [comment]
Here is a screenshot without the tag:

Screenshot 2020-09-22 at 10 26 22

**Review by gitdubz — APPROVED** [review]

**Pull request merged**